src/utils/screenshots.js

```diff
@@ -0,0 +1,65 @@
+const getCustomScreenshots = (ticket, defaultUserAgent) => {
```

Comment:
Could we possibly refactor into a Screenshot Manager object that has a local `defaultUserAgent`? It seems odd to pass a constant we can fetch once into every function.

Reply:
Alternatively, `init` function that fills in a file-scoped variable, like in `db`, with the user agent.